src/renderer/components/EnhancedTable/__tests__/TableRowListenerWrapper.spec.js

```diff
@@ -102,19 +85,6 @@ describe('/renderer/components/EnhancedTable/TableRowListenerWrapper', () => {
         synced: true,
       },
     ],
-    syncedNamespaces: [
-      {
-        cloudUrl: 'https://foobar.com',
-        clusterCount: 4,
-        credentialCount: 4,
-        licenseCount: 1,
-        machineCount: 12,
-        name: 'foobar',
-        proxyCount: 0,
-        sshKeyCount: 2,
-        synced: true,
-      },
-    ],
   });
 
   [disconnectedFakeCloud, connectingFakeCloud, connectedFakeCloud].forEach(
```

Comment:
Since these clouds are really test cases as well as test data, make them consts and localize them to this particular test. I'm just concerned that having them in the outer scope will result in someone writing a test in the future using one of these, having the test modify it, without realizing that they aren't rebuilt with a `beforeEach()` on every test, and then we end-up with a really hard-to-find bug where one test depends on another's output.

So like this:

```suggestion
    (function () {
      const disconnectedFakeCloud = new Cloud(...);
      const connectingFakeCloud = new Cloud(...);
      const connectingFakeCloud = new Cloud(...);
      
      [disconnectedFakeCloud, connectingFakeCloud, connectedFakeCloud].forEach(
        ...
      );
    })();
```

Reply:
Hmm, good solution! Thank you, fixed